package org.forcepower.starcement.bean;


public final class OrderFormDetails {
	String orderFormId = "";
	String userId = "";
	String creditLimit = "";
	String closingStk = "";
	String mrp = "";
	String mrpDrpdwn = "";
	String tradeDiscount = "";
	String addCustomer = "";
	String customerBsnsProspct = "";
	String tdType = "";
	String saleRate = "";
	String saleRateDrpdwn = "";
	String saudaSaleRateDrpdwn = "";
	String attachedPrinter = "";
	String printer_mandetory = "";
	String payment_type = "";
	String mode_of_payment = "";
    String lastUpdateTime = "";
    String tagDistributor = "";
    String sale = "";
    String instruction = "";
    String vat = "";
    String vatDetails = "";
    String branchRDSTransfer = "";
    String vatType = "";
    String tdCalc = "";
    String tdTransType = "";
    
    String vatCalcOn = "";
    
    String saudaDepo = "";
    String saudaCarry = "";
    String saudaRate = "";
    String saudaValue = "";
    String amount 				="";
    
    String mTDValidation		="";
    String mTDCalcBasedOn		="";
    
    String mPremium				="";
    String mPreviousOrder		="";
    String mNewCustomerOtp				="";
    String mCustomerInfoCheck			="";
    String mAddCustomerRouteCreation	="";
    String mOrderType					="";
    String mFreightComponent			="";
    String mTaxType						="";
    String mDestination					="";
    String inputScreenNormal			="";
    String inputScreenSpecial			="";
    String addCustomerDetails			="";
    String addCustomerTradeNonTrade		="";
    String printMedium		="";//printer_type column
    String printerMenu		="";//printer_menu column
    String hintsRemarks		="";
    String hintsRemarksVal		="";
    String addCustomImage		="";
    String distributorRouteEmployeeRelation		="";
    String multiple_UOM 		="";
    String input_screen_planwise 		="";
    String TD_type_input_dropdown 		="";
    String input_screen_planwise_filter1wise  		="";
    String input_screen_price_validation  		="";

    public String getAddCustomerTradeNonTrade() {
		return addCustomerTradeNonTrade;
	}
	public void setAddCustomerTradeNonTrade(String addCustomerTradeNonTrade) {
		this.addCustomerTradeNonTrade = addCustomerTradeNonTrade;
	}
	
	public String getAddCustomerDetails() {
		return addCustomerDetails;
	}
	public void setAddCustomerDetails(String addCustomerDetails) {
		this.addCustomerDetails = addCustomerDetails;
	}
	public String getInputScreenNormal() {
		return inputScreenNormal;
	}
	public void setInputScreenNormal(String inputScreenNormal) {
		this.inputScreenNormal = inputScreenNormal;
	}
	
	public String getInputScreenSpecial() {
		return inputScreenSpecial;
	}
	public void setInputScreenSpecial(String inputScreenSpecial) {
		this.inputScreenSpecial = inputScreenSpecial;
	}
	
	public synchronized String getSaudaValue() {
		return saudaValue;
	}
	public synchronized void setSaudaValue(String saudaValue) {
		this.saudaValue = saudaValue;
	}
	public String getSaudaDepo() {
		return saudaDepo;
	}
	public void setSaudaDepo(String saudaDepo) {
		this.saudaDepo = saudaDepo;
	}
	public String getSaudaCarry() {
		return saudaCarry;
	}
	public void setSaudaCarry(String saudaCarry) {
		this.saudaCarry = saudaCarry;
	}
	public String getSaudaRate() {
		return saudaRate;
	}
	public void setSaudaRate(String saudaRate) {
		this.saudaRate = saudaRate;
	}
	
	public String getVatCalcOn() {
		return vatCalcOn;
	}
	public void setVatCalcOn(String vatCalcOn) {
		this.vatCalcOn = vatCalcOn;
	}
	public String getAmount() {
		return amount;
	}
	public void setAmount(String amount) {
		this.amount = amount;
	}
	public String getVatType() {
		return vatType;
	}
	public void setVatType(String vatType) {
		this.vatType = vatType;
	}
	public String getTdCalc() {
		return tdCalc;
	}
	public void setTdCalc(String tdCalc) {
		this.tdCalc = tdCalc;
	}
	public String getTdTransType() {
		return tdTransType;
	}
	public void setTdTransType(String tdTransType) {
		this.tdTransType = tdTransType;
	}
	public String getBranchRDSTransfer() {
		return branchRDSTransfer;
	}
	public void setBranchRDSTransfer(String branchRDSTransfer) {
		this.branchRDSTransfer = branchRDSTransfer;
	}
	public String getVat() {
		return vat;
	}
	public void setVat(String vat) {
		this.vat = vat;
	}
	public String getVatDetails() {
		return vatDetails;
	}
	public void setVatDetails(String vatDetails) {
		this.vatDetails = vatDetails;
	}
	public String getInstruction() {
		return instruction;
	}
	public void setInstruction(String instruction) {
		this.instruction = instruction;
	}
	public String getSale() {
		return sale;
	}
	public void setSale(String sale) {
		this.sale = sale;
	}
	public String getTagDistributor() {
		return tagDistributor;
	}
	public void setTagDistributor(String tagDistributor) {
		this.tagDistributor = tagDistributor;
	}
	public String getLastUpdateTime() {
		return lastUpdateTime;
	}
	public void setLastUpdateTime(String lastUpdateTime) {
		this.lastUpdateTime = lastUpdateTime;
	}
	public String getPayment_type() {
		return payment_type;
	}
	public void setPayment_type(String payment_type) {
		this.payment_type = payment_type;
	}
	public String getMode_of_payment() {
		return mode_of_payment;
	}
	public void setMode_of_payment(String mode_of_payment) {
		this.mode_of_payment = mode_of_payment;
	}
	public String getPrinter_mandetory() {
		return printer_mandetory;
	}
	public void setPrinter_mandetory(String printer_mandetory) {
		this.printer_mandetory = printer_mandetory;
	}
	public String getAttachedPrinter() {
		return attachedPrinter;
	}
	public void setAttachedPrinter(String attachedPrinter) {
		this.attachedPrinter = attachedPrinter;
	}
	public String getTdType() {
		return tdType;
	}
	public void setTdType(String tdType) {
		this.tdType = tdType;
	}
	public String getSaleRate() {
		return saleRate;
	}
	public void setSaleRate(String saleRate) {
		this.saleRate = saleRate;
	}
	public String getSaleRateDrpdwn() {
		return saleRateDrpdwn;
	}
	public void setSaleRateDrpdwn(String saleRateDrpdwn) {
		this.saleRateDrpdwn = saleRateDrpdwn;
	}
	public String getOrderFormId() {
		return orderFormId;
	}
	public void setOrderFormId(String orderFormId) {
		this.orderFormId = orderFormId;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public String getCreditLimit() {
		return creditLimit;
	}
	public void setCreditLimit(String creditLimit) {
		this.creditLimit = creditLimit;
	}
	public String getClosingStk() {
		return closingStk;
	}
	public void setClosingStk(String closingStk) {
		this.closingStk = closingStk;
	}
	public String getMrp() {
		return mrp;
	}
	public void setMrp(String mrp) {
		this.mrp = mrp;
	}
	public String getMrpDrpdwn() {
		return mrpDrpdwn;
	}
	public void setMrpDrpdwn(String mrpDrpdwn) {
		this.mrpDrpdwn = mrpDrpdwn;
	}
	public String getTradeDiscount() {
		return tradeDiscount;
	}
	public void setTradeDiscount(String tradeDiscount) {
		this.tradeDiscount = tradeDiscount;
	}
	public String getAddCustomer() {
		return addCustomer;
	}
	public void setAddCustomer(String addCustomer) {
		this.addCustomer = addCustomer;
	}
	public String getCustomerBsnsProspct() {
		return customerBsnsProspct;
	}
	public void setCustomerBsnsProspct(String customerBsnsProspct) {
		this.customerBsnsProspct = customerBsnsProspct;
	}
	
	public String getTDValidation() {
		return mTDValidation;
	}
	public void setTDValidation(String tdvalidation) {
		this.mTDValidation = tdvalidation;
	}
	
	public String getTDCalcBasedOn() {
		return mTDCalcBasedOn;
	}
	public void setTDCalcBasedOn(String tdcalcbasedOn) {
		this.mTDCalcBasedOn = tdcalcbasedOn;
	}
	
	public String getPremium() {
		return mPremium;
	}
	public void setPremium(String premium) {
		this.mPremium = premium;
	}
	
	public String getPreviousOrder() {
		return mPreviousOrder;
	}
	public void setPreviousOrder(String previousOrder) {
		this.mPreviousOrder = previousOrder;
	}
	
	public String getNewCustomerOtp() {
		return mNewCustomerOtp;
	}
	public void setNewCustomerOtp(String newCustomerOtp) {
		this.mNewCustomerOtp = newCustomerOtp;
	}
	
	public String getCustomerInfoCheck() {
		return mCustomerInfoCheck;
	}
	public void setCustomerInfoCheck(String customerInfoCheck) {
		this.mCustomerInfoCheck = customerInfoCheck;
	}
	
	public String getAddCustomerRouteCreation() {
		return mAddCustomerRouteCreation;
	}
	public void setAddCustomerRouteCreation(String addCustomerRouteCreation) {
		this.mAddCustomerRouteCreation = addCustomerRouteCreation;
	}
	
	public String getOrderType() {
		return mOrderType;
	}
	public void setOrderType(String orderType) {
		this.mOrderType = orderType;
	}
	
	public String getFreightComponent() {
		return mFreightComponent;
	}
	public void setFreightComponent(String freightComponent) {
		this.mFreightComponent = freightComponent;
	}
	public String getTaxType() {
		return mTaxType;
	}
	public void setTaxType(String taxType) {
		this.mTaxType = taxType;
	}
	public String getDestination() {
		return mDestination;
	}
	public void setDestination(String destination) {
		this.mDestination = destination;
	}
	public String getPrintMedium() {
		return printMedium;
	}
	public void setPrintMedium(String printMedium) {
		this.printMedium = printMedium;
	}
	public String getPrinterMenu() {
		return printerMenu;
	}
	public void setPrinterMenu(String printerMenu) {
		this.printerMenu = printerMenu;
	}
	public String getHintsRemarks() {
		return hintsRemarks;
	}
	public void setHintsRemarks(String hintsRemarks) {
		this.hintsRemarks = hintsRemarks;
	}
	public String getHintsRemarksVal() {
		return hintsRemarksVal;
	}
	public void setHintsRemarksVal(String hintsRemarksVal) {
		this.hintsRemarksVal = hintsRemarksVal;
	}
	public String getAddCustomImage() {
		return addCustomImage;
	}
	public void setAddCustomImage(String addCustomImage) {
		this.addCustomImage = addCustomImage;
	}
	public String getDistributorRouteEmployeeRelation() {
		return distributorRouteEmployeeRelation;
	}
	public void setDistributorRouteEmployeeRelation(String distributorRouteEmployeeRelation) {
		this.distributorRouteEmployeeRelation = distributorRouteEmployeeRelation;
	}

	public String getMultipleUom() {
		return multiple_UOM;
	}
	public void setMultipleUom(String multiple_UOM) {
		this.multiple_UOM = multiple_UOM;
	}

	public String getInputScreenPlanwise()
	{
		return input_screen_planwise;
	}
	public void setInputScreenPlanwise(String input_screen_planwise)
	{
		this.input_screen_planwise = input_screen_planwise;
	}

	public String getTdInputDropDown()
	{
		return TD_type_input_dropdown;
	}
	public void setTdInputDropDown(String TD_type_input_dropdown)
	{
		this.TD_type_input_dropdown = TD_type_input_dropdown;
	}

	public String getInputScreenPlanwiseFilter1Wise()
	{
		return input_screen_planwise_filter1wise;
	}
	public void setInputScreenPlanwiseFilter1Wise(String input_screen_planwise_filter1wise)
	{
		this.input_screen_planwise_filter1wise = input_screen_planwise_filter1wise;
	}

	public String getInputScreenPriceValidation()
	{
		return input_screen_price_validation;
	}
	public void setInputScreenPriceValidation(String input_screen_price_validation)
	{
		this.input_screen_price_validation = input_screen_price_validation;
	}

	public String getSaudaSaleRateDrpdwn() {
		return saudaSaleRateDrpdwn;
	}
	public void setSaudaSaleRateDrpdwn(String saudaSaleRateDrpdwn) {
		this.saudaSaleRateDrpdwn = saudaSaleRateDrpdwn;
	}
	
}
